src/main/java/com/cozyhome/onlineshop/basketcartservice/model/BasketRecord.java

```diff
@@ -0,0 +1,39 @@
+package com.cozyhome.onlineshop.basketcartservice.model;
+
+import com.cozyhome.onlineshop.inventoryservice.model.ProductColor;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity
+@Table(name = "shopping_cart_lines")
```

Comment:
left old name of table for testing needs, will change for production

Reply:
the model package should be moved to basketservice